ega-charts/localega/values.yaml

```diff
@@ -267,6 +267,7 @@ tester:
   run: false
   name: tester
   log: "debug"
+  size: "small"
   repository: nbisweden/localega-tester
-  imageTag: release.v0.0.1
+  imageTag: latest
```

Comment:
We should always try to use a fixed tag. And `latest` builds should also have their own assignable id.

Reply:
Normally yes, this is just a stopgap to be able to run more advanced tests.